.github/workflows/CI.yml

```
@@ -14,36 +14,40 @@ jobs:
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: 4. Copy application.yml
+        env:
+          CREATE_SECRET: ${{ secrets.APPLICATION_YML }}
+          CREATE_SECRET_DIR: src/main/resources
+          CREATE_SECRET_DIR_FILE_NAME: application.yml
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:10: Double quote to prevent globbing and word splitting

(shellcheck)